core/src/main/resources/defaults/blueprints/cdp-sdx-medium-ha-722.bp

```diff
@@ -186,6 +195,11 @@
             ],
             "refName": "solr-SOLR_SERVER-BASE",
             "roleType": "SOLR_SERVER"
+          },
+          {
+            "base": true,
+            "refName": "solr-GATEWAY-BASE",
```

Comment:
is there also not a hbase gateway and master base and a kafkabase and a hdfs base?  Or are they already there?

Reply:
looks like they are already there

Reply:
cools